src/Umbraco.Infrastructure/Persistence/Repositories/Implement/RepositoryBaseOfTIdTEntity.cs

```diff
@@ -132,7 +132,7 @@ protected virtual RepositoryCachePolicyOptions DefaultOptions
         /// </summary>
         /// <remarks>This method is backed by an <see cref="IAppPolicyCache"/> cache</remarks>
         /// <param name="entity"></param>
-        public void Save(TEntity entity)
+        public virtual void Save(TEntity entity)
```

Comment:
Not needed, see previous comment about explicitly implementing HasIdentity.

Reply:
Left in place for now, see response to comment above.